src/controllers/auth/token.js

```diff
@@ -217,6 +217,34 @@ const refreshAccessToken = async (req, res) => {
   }
 };
 
+const removeTokens = async (req, res) => {
+  /*
+    #swagger.tags = ['Auth']
+    #swagger.summary = 'Remove Tokens'
+    #swagger.description = 'Endpoint to remove access and refresh tokens'
+  */
+
+  try {
+    res.clearCookie(authConstants.ACCESS_TOKEN_SUBJECT);
+    res.clearCookie(authConstants.REFRESH_TOKEN_SUBJECT);
+
```

Comment:
@IanMayo, do we need to consider revoking refresh token somehow?

Reply:
@thevahidal Do you have any good strategy to revoke the access of the refresh token?

Reply:
We can introduce a table named `_revoked_refresh_tokens`. This table will have two key fields: refresh_token (containing the JWT refresh token) and expires_at (a datetime field). Whenever a user logs out, we will store their refresh token in this table along with its expiration time.

Additionally, we need to modify the refresh token endpoint to include a check for revoked tokens. During the token refreshing process, we will verify if the provided refresh token exists in the `_revoked_tokens` table. If it does, we will prevent the refresh operation from proceeding.